backend/portfolio42/settings.py

```diff
@@ -25,8 +25,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
-
+ALLOWED_HOSTS = ['0.0.0.0', 'localhost']
```

Comment:
What does this change do?

Reply:
This is allowing explicitly the localhost and 0.0.0.0 to make requests to the API without any kind of restraint.

Some tools and browsers will need you also to allow them explicitly, like insomnia. Usually this is only mandatory for production, but it's a good addition.